src/Symfony/Bridge/Twig/Extension/IntlExtension.php

```diff
@@ -0,0 +1,411 @@
+        'secondary_grouping_size' => NumberFormatter::SECONDARY_GROUPING_SIZE,
+        'significant_digits_used' => NumberFormatter::SIGNIFICANT_DIGITS_USED,
+        'min_significant_digits_used' => NumberFormatter::MIN_SIGNIFICANT_DIGITS,
+        'max_significant_digits_used' => NumberFormatter::MAX_SIGNIFICANT_DIGITS,
+        'lenient_parse' => NumberFormatter::LENIENT_PARSE,
+    );
+    private $numberRoundingAttributes = array(
+        'ceiling' => NumberFormatter::ROUND_CEILING,
+        'floor' => NumberFormatter::ROUND_FLOOR,
+        'down' => NumberFormatter::ROUND_DOWN,
+        'up' => NumberFormatter::ROUND_UP,
+        'halfeven' => NumberFormatter::ROUND_HALFEVEN,
+        'halfdown' => NumberFormatter::ROUND_HALFDOWN,
+        'halfup' => NumberFormatter::ROUND_HALFUP,
+    );
+    private $numberPaddingAttributes = array(
+        'before_prefix' => NumberFormatter::PAD_BEFORE_PREFIX,
+        'after_prefix' => NumberFormatter::PAD_AFTER_PREFIX,
+        'before_suffix' => NumberFormatter::PAD_BEFORE_SUFFIX,
+        'after_suffix' => NumberFormatter::PAD_AFTER_SUFFIX,
+    );
+    private $numberTextAttributes = array(
+        'positive_prefix' => NumberFormatter::POSITIVE_PREFIX,
+        'positive_suffix' => NumberFormatter::POSITIVE_SUFFIX,
+        'negative_prefix' => NumberFormatter::NEGATIVE_PREFIX,
+        'negative_suffix' => NumberFormatter::NEGATIVE_SUFFIX,
+        'padding_character' => NumberFormatter::PADDING_CHARACTER,
+        'currency_mode' => NumberFormatter::CURRENCY_CODE,
+        'default_ruleset' => NumberFormatter::DEFAULT_RULESET,
+        'public_rulesets' => NumberFormatter::PUBLIC_RULESETS,
+    );
+    private $symbols = array(
+        'decimal_separator' => NumberFormatter::DECIMAL_SEPARATOR_SYMBOL,
+        'grouping_separator' => NumberFormatter::GROUPING_SEPARATOR_SYMBOL,
+        'pattern_separator' => NumberFormatter::PATTERN_SEPARATOR_SYMBOL,
+        'percent' => NumberFormatter::PERCENT_SYMBOL,
+        'zero_digit' => NumberFormatter::ZERO_DIGIT_SYMBOL,
+        'digit' => NumberFormatter::DIGIT_SYMBOL,
+        'minus_sign' => NumberFormatter::MINUS_SIGN_SYMBOL,
+        'plus_sign' => NumberFormatter::PLUS_SIGN_SYMBOL,
+        'currency' => NumberFormatter::CURRENCY_SYMBOL,
+        'intl_currency' => NumberFormatter::INTL_CURRENCY_SYMBOL,
+        'monetary_separator' => NumberFormatter::MONETARY_SEPARATOR_SYMBOL,
+        'exponential' => NumberFormatter::EXPONENTIAL_SYMBOL,
+        'permill' => NumberFormatter::PERMILL_SYMBOL,
+        'pad_escape' => NumberFormatter::PAD_ESCAPE_SYMBOL,
+        'infinity' => NumberFormatter::INFINITY_SYMBOL,
+        'nan' => NumberFormatter::NAN_SYMBOL,
+        'significant_digit' => NumberFormatter::SIGNIFICANT_DIGIT_SYMBOL,
+        'monetary_grouping_separator' => NumberFormatter::MONETARY_GROUPING_SEPARATOR_SYMBOL,
+    );
+
+    public function __construct($defaultLocale = null)
+    {
+        $this->defaultLocale = $defaultLocale;
```

Comment:
-1 for the default locale. The intl extension has ``Locale::getDefault()`` for that.